### PR TITLE
resolving coding convention problems

### PR DESCRIPTION
**Pull request by jphuong**
changes made to _version.py, ogh.py, and ogh_meta.py

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -54,9 +54,9 @@ install:
 
 script:
   - if [[ $TEST_TARGET == 'default' ]]; then
-      cp -r tests /tmp && cd /tmp ;
-      pytest -vv tests ;
+      cp -r ogh/tests /tmp && cd /tmp ;
+      pytest -vv test_ogh.py ;
```
Comment (lsetiawan) [inline .travis.yml]: You should keep the tests folder, rather than pytest a specific file

File: ogh/__init__.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
                         print_function,
                         unicode_literals)
 
-from ogh.ogh_meta import *  # noqa
-from ogh.ogh import *  # noqa
+from .ogh_meta import *  # noqa
```
Comment (lsetiawan) [inline ogh/__init__.py:6]: The explicitness of `ogh.ogh_meta` is better than just `.`

File: .travis.yml (hunk 1 above)
Comment (jphuong) [reply .travis.yml]: @lsetiawan 
Should that be coded as follows, to run pytest on files that say 'test' within the tmp folder?

cp -r ogh/tests /tmp && cd/tmp ;
pytest -vv test*

File: ogh/__init__.py (hunk 2 above)
Comment (jphuong) [reply ogh/__init__.py:6]: @lsetiawan 
I would normally agree, but explicit submodule reference was not building. The prefix 'ogh.' within the ogh/__init__.py file was not being recognized when travis CI builds the test environment.

I referred to Ariel's shablona tutorial to make ogh_meta a relative submodule whenever ogh is called. It is building at least.

Comment (jphuong) [reply ogh/__init__.py:6]: hm... perhaps not...

File: .travis.yml (hunk 1 above)
Comment (lsetiawan) [reply .travis.yml]: if you want the specific file to run just do `pytest -vv tests/test_ogh.py`.

**Comment by lsetiawan** [comment]
Could you add another conda install for landlab from landlab channel for now, so at least your tests, run?

**Comment by jphuong** [comment]
@lsetiawan 
The landlab operations were newly introduced, and should not be part of this pull request. I will remove landlab for now.

**Pull request closed without merging**